### PR TITLE
Update: Wishlist

### PR DESCRIPTION
**Pull request by bossmonkey98**
Wishlist update
Apex Store — Mozilla Firefox 19-02-2022 22_19_49
d

### CODE REVIEW

**Review by sanjitsarkar — APPROVED** [review]
Looks good

**Review by humzahasan — APPROVED** [review]
Overall code looks good.

Suggestion : Try to play with the badge component on the card, reason being the black text on red isn't easily readable.

File: components/pages/wishlist/wishlist.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="wishlist.css">
+    <link rel="stylesheet" href="https://apexui.netlify.app/main.css">
```
Comment (dev-prabhat) [inline components/pages/wishlist/wishlist.html:5]: try to put CSS link in single file if possible

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
UI looks clean but used a little darker shade of red because it's bit difficult to read badge on card component

**Review by Muzammil-khan710 — APPROVED** [review]

**Pull request merged**